src/adv.py

```diff
@@ -49,3 +53,34 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+print(textwrap.dedent(
+    f"""\
+    You are in room: {player.current_room.name}
+    {player.current_room.description}!\
+    """))
+
+print(textwrap.dedent('''
+        What would you like to do?
+        [n] Go North [s] Go South [e] Go East [w] Go West
+        [take out <item>] or [return <item>]
+        [drop <item>] Remove item from bag
+        [b] look in bag
+        [q] Quit
+    '''))
+
+while True:
+    choice = input()
+
+    if choice.count(" ") > 1:
+        choice = input("\nMove not allowed.\n")
+    else:
+        action = choice
+
+    if action == "take out" or action == "return":
```

Comment:
You could make this `if action.lower() == "take out"` etc. to account for any letter case passed in through the prompt.